api.json

```diff
@@ -1349,17 +1341,13 @@
             "type": "string",
             "example": "why not to choice physics HL"
           },
-          "postId": {
```

Comment:
post id can be the same across different regions don't they, thus removing post id may not be a wise decision